code/modules/projectiles/gun_system.dm

```diff
@@ -57,6 +57,12 @@
 	var/movement_acc_penalty_mult = 5				//Multiplier. Increased and decreased through attachments. Multiplies the accuracy/scatter penalty of the projectile when firing onehanded while moving.
 	var/fire_delay = 6							//For regular shots, how long to wait before firing again.
 	var/shell_speed_mod	= 0						//Modifies the speed of projectiles fired.
+	/// Determines which humans the gun's shot will pass through based on the victim's ID access list.
+	var/list/gun_iff_signal = null
+	///Determines how fire delay is changed when aim mode is active
+	var/aim_fire_delay = 0
+	///Determines character slowdown from aim mode. Default is 66%
+	var/aim_movement_speed = 6
```

Comment:
This should be something like a "aim speed modifier" where negative values decrease the speed so its more understandable

Reply:
changed